packages/graphql/src/createSchema/NumberScalar.ts

```diff
@@ -0,0 +1,26 @@
+import { GraphQLScalarType } from "graphql";
+import { Kind } from "graphql/language";
+
+// TODO: Need to check whether user can send number as string
+// And do we want to parse it in that case or use null
+const getNumber = (value: any): any => {
+    if (typeof value !== "number") {
+        return null;
+    }
+
+    return value;
+};
+
+export const Number = new GraphQLScalarType({
+    name: "Number",
+    description: "A custom input type to be used with numbers. Supports Int and Float.",
+    serialize: getNumber,
+    parseValue: getNumber,
+    parseLiteral: ast => {
+        if (ast.kind === Kind.INT || ast.kind === Kind.FLOAT) {
+            return ast.value;
+        }
+
+        throw new Error("Invalid Number value!");
```

Comment:
For example, when you send `false` instead of a string to a `String` field, the error message would be the following:

> Expected type String, found false.

To be a bit more uniformed, lets change the error message to:
> `"Expected type Number, found {value}."`.